### PR TITLE
Add dk lang

### PR DESCRIPTION
**Pull request by LDalby**
This addresses #1985

### CODE REVIEW

**Comment by hadley** [comment]
@thomasp85 would you mind proofing this?

**Comment by LDalby** [comment]
@thomasp85 I case you end up proofing this - I didn't translate "contractor" because I couldn't come up with a good translation in this context. 
I thought that "underleverandør" might work, but then I looked at the German translation which had "Auftragnehmer" (DK: ordremodtager) for this. The direct translation of "contractor" to Danish is everything from "bygherre" to "ordregiver" which is the direct opposite to the German translation...
I was not able to figure out where on the rendered pkgdown site "contractor" would show up, so I just left it untranslated.
Perhaps you can come up with something or perhaps point me to the place where it is used?

**Comment by hadley** [comment]
@LDalby contractor is one of the author "roles" that can appear in the description. R's documentation says: 

> Use for authors who have been contracted to write (parts of) the package and hence do not own intellectual property.

I don't know if that helps much, but it's rarely used so it doesn't matter to much if the translation isn't perfect.

**Comment by LDalby** [comment]
@hadley Thanks, that clarified the context, so now I think it is good enough.

File: po/R-dk.po [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: dk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: build-404.R:11
+msgid "Page not found (404)"
+msgstr "Side ikke fundet"
```
Comment (thomasp85) [inline po/R-dk.po:15]: Shouldn't we keep the error code?

File: po/R-dk.po [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+msgid "All vignettes"
+msgstr "Alle vignetter"
+
+#: build-footer.R:29
+msgid "Developed by"
+msgstr "Udviklet af"
+
+#: build-footer.R:34
+msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
+msgstr "Hjemmeside bygget med <a href=\"%s\">pkgdown</a> %s."
```
Comment (thomasp85) [inline po/R-dk.po:35]: "Hjemmeside" sounds a bit '90s 🙂. Maybe just "Side"? I have no strong feelings about this however

File: po/R-dk.po [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+msgid "Releases"
+msgstr "Udgivelser"
+
+#: build-news.R:285
+msgid "CRAN release: %s"
+msgstr "CRAN udgivelse: %s"
+
+#: build-reference-index.R:19
+msgid "Function reference"
+msgstr "Funktionsreference"
```
Comment (thomasp85) [inline po/R-dk.po:167]: Maybe "Funktionsoversigt"? but no strong preference for either

**Review by thomasp85 — APPROVED** [review]
By and large its good - I have a couple of comments/suggestions but nothing serious

File: po/R-dk.po (hunk 1 above)
Comment (LDalby) [reply po/R-dk.po]: Yes - think I overlooked that one.

File: po/R-dk.po (hunk 2 above)
Comment (LDalby) [reply po/R-dk.po]: Ha, ha! I totally agree 😃

**Comment by LDalby** [comment]
Thanks @thomasp85!
I've modified in accordance with the suggestions.

**Comment by thomasp85** [comment]
Great - LGTM @hadley

**Comment by hadley** [comment]
Thanks!

**Pull request merged**